server/src/main/java/org/apache/druid/query/lookup/MapLookupExtractorFactory.java

```diff
@@ -145,15 +145,15 @@ public MapLookupIntrospectionHandler(Map<String, String> map)
     @Produces(MediaType.APPLICATION_JSON)
     public Response getKeys()
     {
-      return Response.ok(map.keySet().toString()).build();
+      return Response.ok(map.keySet()).build();
```

Comment:
This response structure aligns with what is already documented: https://druid.apache.org/docs/latest/querying/lookups/#introspect-a-lookup